spark/dl/src/main/scala/com/intel/analytics/bigdl/transform/vision/image/Convertor.scala

```diff
@@ -117,3 +124,80 @@ object MatToFloats {
     outKey: String = ImageFeature.floats): MatToFloats =
     new MatToFloats(validHeight, validWidth, validChannels, meanRGB, outKey)
 }
+
+class FloatsToTensor[T: ClassTag](toRGB: Boolean = false,
```

Comment:
Shy do we need a separate FloatsToTensor?

Reply:
Because I want to make ImageFrameToSample more general, ImageFrameToSample can accept multiple input keys and each key maps a tensor.

Or is it better to make this transformer MatToTensor? (instead of MatToFloat->FloatToTensor)

Reply:
Yes, MatToTensor might be better